mat64/dense.go

```diff
@@ -379,7 +379,9 @@ func (m *Dense) Clone(a Matrix) {
 
 // Copy makes a copy of elements of a into the receiver. It is similar to the
 // built-in copy; it copies as much as the overlap between the two matrices and
-// returns the number of rows and columns it copied.
+// returns the number of rows and columns it copied. If a aliases the receiver
+// and is a transposed Dense or Vector, with a non-unitary increment, Copy will
+// panic.
```

Comment:
I realize that it can be done, but is there a particular case that makes copy with transpose and aliasing useful? The rule is more clear if it's "aliasing is okay except for transposes", and not "aliasing is okay except for transposes except when m has one dimension equal to one and hasn't been viewed". A function can't tell in general if the matrix has been viewed, and so it seems more prone to error to allowing certain special cases (i.e. this function worked fine until I sent in a viewed matrix)